packages/grafana-ui/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactNode } from 'react';
+
+import { t } from '../../utils/i18n';
+import { Box } from '../Layout/Box/Box';
+import { Stack } from '../Layout/Stack/Stack';
+import { Text } from '../Text/Text';
+
+import { GrotNotFound } from './GrotNotFound/GrotNotFound';
+
+interface Props {
+  /**
+   * Provide a button to render below the message
+   */
+  button?: ReactNode;
+  hideImage?: boolean;
+  /**
+   * Override the default image for the variant
+   */
+  image?: ReactNode;
+  message?: string;
+  variant: 'search';
```

Comment:
Can we document all the props? This will show them in the props table with the description text.